builtins/amp-img.md

```diff
@@ -144,6 +144,9 @@ If you want your image to scale as the window is resized but up to a maximum wid
 1. Set `layout=responsive` for `<amp-img>`.
 2. On the container of the image, specify the `max-width:<max width to display image>` CSS attribute.  Why on the container?  An `amp-img` element with `layout=responsive` is a *block-level* element, whereas, `<img>` is *inline*. Alternatively, you could set `display: inline-block` in your CSS for the amp-img element.
 
+#### Understanding responsive vs intrinsic layout
+
+Both the `responsive` and `intrinsic` layouts create an image that will scale automatically. The main difference is that the `intrinsic` layout uses an SVG image as it's scaling element.  This will make it behave in the same way as a standard html image while retaining the benefit of the browser knowing the image size on initial layout. The `intrinsic` layout will have an intrinsic size and will inflate a floated `div` until it reaches either the natural image size or a CSS constraint like `max-width`. The `responsive` layout will render 0x0 in a floated `div` because it takes its size from the parent which in a float has no natural size.
```

Comment:
I found this confusing.  Can you clarify:
- "uses an SVG image" - is that internal or does the developer have to specify it?
- "uses an SVG image as it's scaling element" - if `intrinsic` uses an SVG image, what does responsive use for scaling?
- when shouldn't you use `intrinsic` over `responsive` ?
- "The `intrinsic` layout will have an intrinsic size" - specified by whom, from where?  Clarify plz.
-  "CSS constraint like `max-width`." - specified on the parent yes?  Please add for clarification, otherwise the reader might imply they can use max-width on the amp-img.

Reply:
I'll take another stab at this.  The max-width can be on the element or a parent.  I think I might need some pictures and example code to make this clear.  As to when you should use intrinsic, the answer is all the time, The only reason to not make responsive behave this way is that it would be a breaking change.